piano.yml

```diff
@@ -338,7 +340,8 @@ Layer:
           FROM
             osm_admin
           WHERE
-            admin_level = 3 OR admin_level = 4
+            (admin_level = 3 OR admin_level = 4)
+            AND osm_id < 0
```

Comment:
DB indexes will need an update I think.
Maybe it's time to create a generalized_table instead? I guess it's a quite small dataset given those filters.